tests/Routes/Wiki/SettingUpdateTest.php

```diff
@@ -93,10 +100,17 @@ public function provideValidSettingsBadValues() {
         yield [ 'wwWikibaseStringLengthMultilang', 12 ];
         yield [ 'wikibaseFedPropsEnable', 'foo' ];
         yield [ 'wikibaseManifestEquivEntities', 'foo' ];
-        yield [ 'wikibaseManifestEquivEntities', json_encode( [ 'P1', 'P2' ] ) ];
-        yield [ 'wikibaseManifestEquivEntities', json_encode( [ 'P1' => 'P2', 'P3' => 'aa' ] ) ];
-        yield [ 'wikibaseManifestEquivEntities', json_encode( [ 'P1' => 'P2', 'aa' => 'Q2' ] ) ];
-        yield [ 'wikibaseManifestEquivEntities', json_encode( [ 'P1' => 'P2', 'P10' => 'Q2' ] ) ];
+        
+        // props without mapping
+        yield [ 'wikibaseManifestEquivEntities', json_encode( [ 'properties' => [ 'P1', 'P2' ], 'items' => []] ) ];
+        // invalid property id
+        yield [ 'wikibaseManifestEquivEntities', json_encode( [ 'properties' => [ 'P1' => 'P2', 'P3' => 'aa' ], 'items' => []] ) ];
+        // invalid property type
```

Comment:
Not sure if this one ends up being invalid property type or invalid key?
I think I'll add one more test case and then merge it